### PR TITLE
Isort has stopped working

### PR DESCRIPTION
**Pull request by MauriceHendrix**
As you can see from the non-pinned python 3.6 test the isort config has stopped working.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -25,3 +25,4 @@ script:
   - isort --verbose --check-only --diff --recursive cellmlmanip
```
Comment (MauriceHendrix) [inline .travis.yml]:
```suggestion
  - isort --verbose --check-only --diff
```

Comment (MauriceHendrix) [reply .travis.yml]: on codegen this seemed to fix the issue

**Comment by MichaelClerx** [comment]
Seems to say 0 files changed now?

**Comment by MauriceHendrix** [comment]
Yes the change is in the suggestion If I would have applied it I'm sure you guys would not be able to see why it's needed and would have rejected the proposed change.

**Comment by MauriceHendrix** [comment]
> Github seems to say 0 files changed now in this PR?

Yes the change is in the suggestion If I would have applied it I'm sure you guys would not be able to see why it's needed and would have rejected the proposed change.

File: .travis.yml [hunk 2]
```diff
@@ -25,3 +25,4 @@ script:
...
 after_success:
   - codecov
+
```
Comment (MauriceHendrix) [inline .travis.yml:28]:
```suggestion
```

File: .travis.yml (hunk 1 above)
Comment (MauriceHendrix) [inline .travis.yml:25]:
```suggestion
  - isort --verbose --check-only --diff
```

Comment (MauriceHendrix) [reply .travis.yml:25]: on codegen this seemed to fix the issue

**Comment by jonc125** [comment]
So I would expect a PR like this to explain exactly what has stopped working and some indication of why, along with the actual fix. You could push one commit that demonstrates the broken behaviour, and then one that fixes it.

In this case the issue is that on Python 3.6+ the latest isort 5.0 release gets used, which as a major release contains backwards incompatibilities. See https://github.com/timothycrosley/isort/blob/develop/CHANGELOG.md#500-penny---july-4-2020 for details, in particular the note about the removal of the `--recursive` flag. It's not obvious from the short Travis result that there is a breakage since the test that uses the latest versions of all dependencies is an allowed failure (so we don't need to hold up minor PRs while fixing forward compatibility).

Now, on Python 3.5 we might want to keep the argument, since recursive processing is desired in general, but we can get away without it since we only have a single depth folder. This would break if we removed the `cellmlmanip` argument though. If we want import sorting in tests and setup.py we could use `isort --verbose --check-only --diff cellmlmanip tests setup.py` instead.

**Comment by MauriceHendrix** [comment]
> So I would expect a PR like this to explain exactly what has stopped working and some indication of why, along with the actual fix. You could push one commit that demonstrates the broken behaviour, and then one that fixes it.

Sigh! :no_mouth:

Doesn't even work!

**Comment by jonc125** [comment]
We can probably just remove the `not_skip` item from setup.cfg?

**Comment by MauriceHendrix** [comment]
> We can probably just remove the `not_skip` item from setup.cfg?

that works

**Pull request merged**